### PR TITLE
Document how to use conversion classes

### PR DESCRIPTION
**Pull request by WardLT**
Checklist:
- [x] Include them in the API documentation
- [x] Add an "advanced topic" on them to online estimation
- [x] Explain the reasons why each type of class might be used

### CODE REVIEW

**Comment by WardLT** [comment]
@victorventuri , any more comments before we merge?

File: docs/estimators/online/conversions.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+The :meth:`~moirae.estimators.online.filters.distributions.MultivariateRandomDistribution.convert` method of
+a :class:`~moirae.estimators.online.filters.distributions.MultivariateRandomDistribution`
+employs sample and covariance conversions to produce a new distribution in the desired coordinate system.
+
+Selecting a Conversion Operator
+-------------------------------
+
+Coordinate systems available for models may be problematic for many reasons, and we provide different filters for each:
+
+- *Disparate Scales* can be normalized with the :class:`~moirae.estimators.online.filters.conversions.LinearConversionOperator`.
```
Comment (victorventuri) [inline docs/estimators/online/conversions.rst:36]: Can we aadd the `~moirae.estimators.online.filters.conversions.AbsoluteValueConversionOperator` somewhere here?

**Review by victorventuri — APPROVED** [review]
Looks good, just made one comment about the absolute value

File: docs/estimators/online/conversions.rst (hunk 1 above)
Comment (WardLT) [reply docs/estimators/online/conversions.rst:36]: Yep. It's on the next one below, though I'll fix the name

**Pull request merged**